Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM elixir:1.9-alpine as builder
+
+LABEL author "Emil Bostijancic <emil.bostijancic@gmail.com>"
```

Comment:
Oh, I didn't know we could add authors to Dockerfile. What are best practices around this, do we list all authors that contribute to the project itself or authors that contribute to just Dockerfile? Should we keep the author? Should we add new author tags as people contribute?

Reply:
I would say the best practice would be to leave the author out, I have just copy and pasted the Dockerfile from another Elixir based project.